backend/src/main/kotlin/org/genspectrum/dashboardsbackend/model/triggerevaluation/TriggerEvaluator.kt

```diff
@@ -23,12 +23,54 @@ class TriggerEvaluator(
 
         val dateFilter = computeDateFilter(subscription)
```

Comment:
They will be overwritten (since we do `filter + dateFilter` here). If it were the other way round, the user input would overwrite the date window filters.